### PR TITLE
Return token verification failed message if so

### PR DESCRIPTION
**Pull request by ganesanarun**

### CODE REVIEW

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/in/projecteka/consentmanager/SecurityConfiguration.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=5868799&bid=19209425)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/main/java/in/projecteka/consentmanager/clients/ErrorMap.java  1
- src/main/java/in/projecteka/consentmanager/SecurityConfiguration.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=5868799&bid=19209425)

File: src/main/java/in/projecteka/consentmanager/clients/ErrorMap.java [hunk 1]
```diff
@@ -2,33 +2,39 @@
...
-            put(3405, ErrorCode.OTP_INVALID);
-            put(3406, ErrorCode.OTP_EXPIRED);
-            put(3413, ErrorCode.INVALID_LINK_REFERENCE);
-            put(3507, ErrorCode.FAILED_TO_GET_LINKED_CARE_CONTEXTS);
-            put(3409, ErrorCode.DUPLICATE_DISCOVERY_REQUEST);
-            put(3407, ErrorCode.DISCOVERY_REQUEST_NOT_FOUND);
+public class ErrorMap {
 
+    //hip
+    private static final Map<Integer, ErrorCode> ERROR_CODE_MAP = of(3404, ErrorCode.NO_PATIENT_FOUND,
```
Comment (leelanakka) [inline src/main/java/in/projecteka/consentmanager/clients/ErrorMap.java:21]: proper formatting?

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/main/java/in/projecteka/consentmanager/clients/ErrorMap.java  1
- src/main/java/in/projecteka/consentmanager/SecurityConfiguration.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=5868799&bid=19209425)

**Pull request merged**